### PR TITLE
:fire: WFP-813: Remove beta branding

### PR DESCRIPTION
**Pull request by katie-cawthorne**

### CODE REVIEW

**Review by carlov20 — APPROVED** [review]
👍

File: app/routes/dashboard.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ module.exports = function (router) {
       throw new Error('Only available for National Level')
     }
     return getDashboardFile(fileId).then(function (dataStream) {
-      res.set('Content-disposition', 'attachment; filename=BETA-' + fileId)
+      res.set('Content-disposition', `attachment; filename=${fileId}`)
```
Comment (markreesmoj) [inline app/routes/dashboard.js:80]: 👍

File: app/services/get-export-csv.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ module.exports = function (organisationLevel, result, tab) {
 
   const csv = getCsv(organisationLevel, result, tab, fields, fieldNames)
 
-  return { filename: `BETA-${filename}`, csv: csv }
+  return { filename: filename, csv: csv }
```
Comment (markreesmoj) [inline app/services/get-export-csv.js:59]: `return { filename, csv }`

Might as well go the whole hog

Comment (katie-cawthorne) [reply app/services/get-export-csv.js:59]: Yep, agree

**Review by markreesmoj — APPROVED** [review]
🥇

**Pull request merged**